sparse_ho/algo/implicit.py

```diff
@@ -39,20 +37,21 @@ def get_beta_jac_v(
 
 def get_beta_jac_t_v_implicit(
         X, y, log_alpha, get_v, mask0=None, dense0=None, tol=1e-3,
-        model="lasso", sk=False, max_iter=1000, sol_lin_sys=None):
+        model="lasso", sk=False, max_iter=1000, sol_lin_sys=None,
+        tol_lin_sys=1e-6, max_iter_lin_sys=100):
     alpha = np.exp(log_alpha)
-    n_samples, n_features = X.shape
-
     mask, dense, _ = get_beta_jac_iterdiff(
```

Comment:
this may deserve a comment: we call iterdiff inside implit. My understanding is that it's because we only want beta (mask, dense, other return is discarded, and now I see  `compute_jac` = False which explains), but it surprises the reader a bit

Reply:
I changed `get_beta_jac_iterdiff` for `compute_beta`